lib/themes/dosomething/paraneue_dosomething/scss/patterns/_card.scss

```diff
@@ -42,6 +42,7 @@
 }
 
 
+// Soon to be deprecated in favor of signup-card.
```

Comment:
its 6 minutes later & github still shows this outdated comment. @github please fix.